style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  padding: 0;
+  margin: 0;
+  font-family: "Inter", sans-serif;
+  height: auto;
+  background-image: url(images/header-illsutration-mobile@2x.png);
+  background-repeat: no-repeat;
+  background-size: cover, auto;
+}
+
+.bar {
+  width: 35px;
+  height: 5px;
+  background-color: black;
+  margin: 6px;
+}
+
+.container {
+  padding-left: 1rem;
+  padding-right: 1rem;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+nav {
+  display: flex;
+  justify-content: end;
+}
```

Comment:
You have done great by removing the menu items on mobile version, kindly adjust the background image not to overlap with the hamburger image.

Your design:
![image](https://user-images.githubusercontent.com/20554583/236623528-b2fa2f2e-9755-4fdc-9346-27ca3ab49b43.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/236623537-614d2e9e-451c-4e8c-8bfc-6c035423f23f.png)